lib/cocoapods/target/aggregate_target.rb

```diff
@@ -193,51 +193,21 @@ def framework_paths_by_config
         relevant_pod_targets = pod_targets.select do |pod_target|
           pod_target.include_in_build_config?(target_definition, config)
         end
-        framework_paths_by_config[config] = relevant_pod_targets.flat_map do |pod_target|
```

Comment:
This method is now simplified. Moved most of it into `framework_paths` in `pod_target.rb` because when we integrate test targets we do not have an aggregate target but still want the framework paths.